scripts/superdesk-workspace/content/content.js

```diff
@@ -560,7 +565,13 @@
                  * @description label returns the display name for a key.
                  */
                 scope.label = function(id) {
-                    return labelMap[id];
+                    if (labelMap.hasOwnProperty(id)) {
+                        return labelMap[id];
+                    }
+
+                    console.error('could not find label for ' + id +
```

Comment:
imo would be better to use `console.warn` here, and it shouldn't show the warning to users - they might not be able to do much about it.